install/resources/mybb_theme.xml

```diff
@@ -5509,22 +5509,24 @@ if(use_xmlhttprequest == "1")
 </table>
 </fieldset>]]></template>
 		<template name="member_register_stylebit" version="120"><![CDATA[<option value="{$style['sid']}">{$style['name']}</option>]]></template>
-		<template name="member_resendactivation" version="120"><![CDATA[<html>
+		<template name="member_resendactivation" version="1819"><![CDATA[<html>
```

Comment:
This needs bumping to `1820` since we released 1.8.19 already.

Reply:
Updated